### ISSUE TITLE
there is a way to use  with a  HTTP proxy  ?

### ISSUE THREAD

**Issue by MatiasL13** [fireice-uk/xmr-stak-cpu#120]
it's possible?

**Comment by fireice-uk** [comment]
Not really. HTTP is inherently uni-directional (client requests a page, server serves it). Stratum traffic is bi-directional. HTTP long polling is an inherently bad way to mine.

**Comment by xmrig** [comment]
If HTTP proxy support CONNECT method it can pass any TCP traffic.

**Comment by JokerGermany** [comment]
I used it with minerd, i have only http, https and ftp proxy.
Would appreciate proxy, because the vpn isn't stable

**Comment by ruzickap** [comment]
For example [tpruvot/cpuminer-multi](https://github.com/tpruvot/cpuminer-multi) is working fine with http proxy :+1:.

**Comment by viniciusleterio** [comment]
has any way to use this miner with proxy??

**Comment by kth5** [comment]
Tsocks does the trick on Linux:
> http://tsocks.sourceforge.net/
